### PR TITLE
{vis}[foss/2016b] gnuplot v5.0.5

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2144.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2bbde876b3b7a3df299bcc7d638ee90a for a full test report.

**Comment by boegel** [comment]
@verdurin conflicts?

**Comment by verdurin** [comment]
@boegel ugh. Must admit I'm still getting used to the boegelbot output.

What's the best way forward here?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (1 easyconfigs in this PR)
node2543.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/605620a8831ef7f30ae20ebb8f6ae413 for a full test report.

File: easybuild/easyconfigs/g/gnuplot/gnuplot-5.0.5-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/gnuplot/files', 'download')]
+
+dependencies = [
+    ('cairo', '1.14.6'),
+    ('libjpeg-turbo', '1.5.0'),
+    ('libpng', '1.6.26'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/gnuplot/gnuplot-5.0.5-foss-2016b.eb]: @verdurin you'll need to stick to 1.6.24 here to resolve the conflict

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2663.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ebd1af50e2d373ea0a3ca567a75b5097 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2145.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3b7976d1ed094ea530db87429967d35e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**